Branch-SDK/Branch-SDK/Branch.m

```diff
@@ -1650,7 +1639,9 @@ - (void)serverCallback:(BNCServerResponse *)response {
                 }
             }
             [self handleFailure:[self.requestQueue size]-1];
-        } else if (status != 200) {
+        }
+        // Anything else not between 200 and 400 indicates a server error or iOS error (like connectivity)
+        else if (status < 200 || status > 399) {
```

Comment:
Those are all considered "successful" and fall into the appropriate handler below this block.